pkg/autoscaling/calculation.go

```diff
@@ -288,7 +288,11 @@ func getScaledTiKVGroups(informer core.StoreSetInformer, healthyInstances []inst
 func getScaledTiDBGroups(informer tidbInformer, healthyInstances []instance) []*Plan {
 	planMap := make(map[string]map[string]struct{}, len(healthyInstances))
 	for _, instance := range healthyInstances {
-		tidb := informer.GetTiDB(instance.address)
+		tidb, err := informer.GetTiDB(instance.address)
+		if err != nil {
+			// TODO: error handling
```

Comment:
What should we do to handle this kind of error?

Reply:
I think we could package the error message and tell the caller(deploy layer) the error reason in response.